course/meetings/eeb-marty-2017-04-13.md

```diff
@@ -0,0 +1,61 @@
+
+- Ecology material
+	- It's mostly up to us what we want to include, and I think Martin will be pretty hands-off about what we choose. 
+	- Do some blackboard analysis of simple ecology models:
+		- start with 1D models (i.e. logistic, both continuous and discrete), show phase portraits, local approximation & linearization, plot dN/dt vs. N, etc.
+		- good 2D examples include Lotka-Volterra, competition models
+	- a good exercise could be fitting the Ricker model to some timeseries data - this can be recast as a simple linear regression problem to get the growth rate and carrying capacity, but the model itself has interesting behaviours including several periodicities of limit cycle and chaos. It's also easy (?) to add stochasticity to the model if we want.
+	- Eigenvalues might be hard, maybe doing them computationally and explaining what they mean is good enough.
+	- They don't have to be able to do a full stability analysis by the end of the course - that might be too lofty of a goal. That's an entire course's worth of material.
+	- Need to start from the very beginning with this stuff - they won't know about differential equations - most they've seen is 1D models analyzed graphically on the board (i.e. logistic growth with fixed points and stability intuition from plot). They might have seen a bifurcation diagram, but again, no math.
+	- Some ecology tidbits (mostly for Madeleine's reference):
+		- they discuss in 3rd year logistic growth, the Allee effect, the spruce bugworm model circa 1970s (dN/dt = G(N)-H(N)), G(N) is logistic growth, H(N) is type III predator functional response
+		- functional responses: type I is linear, type II is saturating (Michaelis-Menten type), type III is sigmoidal. Illustrate types with analogy of you going to an all-you-can-eat buffet - what does your response look like? Plots are rate of eating of 1 predator vs. prey abundance. 
+	
+- Course structure and logistics:
+	- Martin will **not** give any lectures, he is only there to help with admin stuff. 
+	- Room has been booked: it's the CQuest (?) room, a computer lab, for both days. We can go and check it out at some point. It has a projector and screen.
+	- We will communicate with IT to discuss the software the computers need: R, RStudio, Git, R packages? Saving permissions?
+	- We can bring in guest lecturers if we want
+	- Unstructured project work time is an acceptable thing to have in place of lecture later in the course.
+	- We can monitor enrolment over the summer, but it won't be a good indicator really until a few weeks into the course. 
+	- Avoid a course website that isn't Blackboard if possible - best to do all dissemination of materials and communication on there unless it's way too clunky for something.
+	- Gather student feedback early and often - we can't change the assessment scheme once it's set, but we can change the content and length of assignments. 
```

Comment:
I made a note that it might be useful to have the survey worth some minor grades to encourage completion. Especially to gauge student knowledge of topics we intend to cover.